File: Com.Fubei.OpenApi.Vb/Biz/AgentApi/Model/Response/ACreateFixedQrcodeResultEntity.vb
Imports Newtonsoft.Json

Namespace Biz.V2.Model.Response
    
    ' 定额码请求响应实体
    Public Class ACreateFixedQrcodeResultEntity
        <JsonProperty("qrcode_url")>
        Public Property QrcodeUrl As String

        <JsonProperty("merchant_order_sn")>
        Public Property MerchantOrderSn As String
    End Class
End Namespace

File: Com.Fubei.Api.OpenApi/Biz/AgentApi/Model/Response/ACreateFixedQrcodeResultEntity.cs
using Newtonsoft.Json;

namespace Com.Fubei.Api.OpenApi.Biz.AgentApi.Model.Response
{
    public class ACreateFixedQrcodeResultEntity
    {
        [JsonProperty("qrcode_url")]
        public string QrcodeUrl { get; set; }

        [JsonProperty("merchant_order_sn")]
        public string MerchantOrderSn { get; set; }
    }
}
